signac/contrib/project.py

```diff
@@ -629,25 +896,42 @@ def find_jobs(self, filter=None, doc_filter=None):
         statepoints, whereas the `doc_filter` argument compares against job
         document keys.
 
-        :param filter: A mapping of key-value pairs that all
-            indexed job statepoints are compared against.
-        :type filter: Mapping
-        :param doc_filter: A mapping of key-value pairs that all
-            indexed job documents are compared against.
-        :type doc_filter: Mapping
-        :yields: Instances of :class:`~signac.contrib.job.Job`
-        :raise TypeError: If the filters are not JSON serializable.
-        :raises ValueError: If the filters are invalid.
-        :raises RuntimeError: If the filters are not supported
+        Parameters
+        ----------
+        filter : Mapping
+            A mapping of key-value pairs that all
+            indexed job statepoints are compared against. (Default value = None)
+        doc_filter : Mapping
+            A mapping of key-value pairs that all
+            indexed job documents are compared against. (Default value = None)
+
+        Yields
+        ------
+        Instances of :class:`~signac.contrib.job.Job`
+
```

Comment:
This should be removed in favor of the `Returns` section. This isn't a generator itself, but it returns one.
```suggestion
```